### ISSUE TITLE
Spawn That - Issue with Valheim Update Patch 0.212.6

### ISSUE THREAD

**Issue by ValhallaCommunity** [ASharpPen/Valheim.SpawnThat#177]
[ServerLogOutput.log](https://github.com/ASharpPen/Valheim.SpawnThat/files/10687246/ServerLogOutput.log)
[LogOutput.log](https://github.com/ASharpPen/Valheim.SpawnThat/files/10687247/LogOutput.log)

Hey! 👋 

With the Valheim release yesterday, I think many servers are experiencing an issue with Spawn That's Spawn Method. It's impacting a few other mods such as Creature Level and Loot Control & KG's Marketplace. I attached the log files for the details.

- Logoutput.log is client side
- Serverlogoutput is server side

Appreciate you taking a look at this and hope to see an update soon!

**Comment by ASharpPen** [comment]
I believe your problem is the outdated Creature Level and Loot Control mod. You need the newer version for it to handle the new valheim patch or it will cause an error that takes Spawn That with it.

**Comment by ValhallaCommunity** [comment]
I'll test the update in the morning. 

The CLLC update, two weeks ago, was only for Epic Loot. I can update it but it hasn't been updated for the valheim patch. I see other servers with CLLC without spawn that working fine. 

Spoke with KG on why marketplace mod wasn't working. He mentioned that it appeared to be spawn that causing the issue.

**Comment by ValhallaCommunity** [comment]
You are right, that fixed the issue! I have a documentation update request :)

I don't know how to get the world spawner data. The recent Valheim patch update enabled Seekers, Seeker broods, and ticks to spawn in the meadows if the mistland queen is killed (similar to the goblins world spawner if Yag is killed). I rely on the documentation to turn those spawns off, could you update that documentation with that data when you have a chance?

Also, if you have a Discord for mod support would love that link! I can close issue, just wanted to see response on the documentation first. Thanks again!

**Comment by ASharpPen** [comment]
The world spawner data is just me uploading the file anyone can generate with Spawn That. I will try to remember and update it on next push, but in the meantime, you can check out this [tutorial ](https://asharppen.github.io/Valheim.SpawnThat/file-configs/1.1.0/tutorials/finding-prefab-names.html#31-world-spawns) for how to make it yourself.

For the discords, I don't have any. I am in several though, such as [Valheim Modding](https://discord.gg/jtuAZ7NM) and [Jötunn](https://discord.com/channels/829466376875933756/829466376875933759)
